### ISSUE TITLE
Suggestion: drop Track 77 OCR (OCR2 is fine)

### ISSUE THREAD

**Issue by edale2** [Insidious611/DancingMadFF6#39]
I'd drop track 77 OCR from the pack, OCR2's track 77 is great, but there's a problem with the OCR one.

The track is 'Fanatics' and plays at the Fanatic's Tower, and there's minor problems there (specifically the LONG mostly silent intro that doesn't actually get to the music until you're 1/2 way up the second section of the tower).

I actually was going to edit the intro out myself when I actually started going through the Fanatic's Tower, when I remembered qwertymodo's edit, checked that, and sure enough, he did the _exact_ edits I was planning on, lol. So I just popped that in, and that was that.

---

The thing is, it _also_ plays at the speeches right before the Kefka fight.

Remember a few weeks ago I asked about the strange track that played during Kefka's rants? You thought I meant the weird sound effects? I was talking abut the actual music that I could faintly hear behind that, I never actually had Kefka's speech last long enough to get to the actual music of that piece to play.

When I got to Kefka's intro dialogue just now (with OCR2's track 77), and the music started instantly.... Yea, the scene's not the same without the music actually playing.

---

Basically, OCR-77's excessively long/quiet intro kill the dramatic tension of the beginning of the final confrontation with Kefka; to the point that it feels like an entirely different scene.

**Comment by edale2** [comment]
I should say I never had that part of the speeches last long enough for the actual music to play, because it switches over to Kefka's Theme part way through.

**Comment by Dizzy611** [comment]
If I  dropped tracks completely for fitting poorly we'd not have like half of the OCRemix tracks.

That said I'll check out what you're talking about, at the very least it may warrant a change in my recommended list

**Comment by edale2** [comment]
It's not so much it fits poorly, it's that it doesn't get a chance to play at all in one of the most dramatic moments of the game.

**Comment by Dizzy611** [comment]
Okay when doing fanatics tower I definitely saw what you mean

**Comment by edale2** [comment]
The fanatics tower isn't even that bad compared to the Kefka pre-final-fight-banter.

**Comment by Dizzy611** [comment]
Okay so, I've done this the easy way (easy because it doesn't involve editing a number of installer data files, which I don't want to do because I consider the installer "frozen" for release atm), by overwriting the track 77 in new/OCR with the one from new/OCR2.

That should fix this for new installs. :)

**Comment by edale2** [comment]
Pretty much what I figured you'd do. Easiest solution.
